docs/azure/migration/appmod/working-with-assessment.md

```diff
@@ -0,0 +1,139 @@
+
+- Example one: you'd like to migrate your apps to Azure but haven't decided on the target compute service yet.
+```json
+{
+  "appcat": {
+    "target": "Any"
+  }
+}
+```
+
+- Example two: you'd like to migrate your apps to App Service Linux and want to understand what are the issues to be fixed.
+```json
+{
+  "appcat": {
+    "target": "AppService.Linux"
+  }
+}
+```
+
+After running an assessment, the interactive dashboard opens automatically, providing comprehensive analysis results. When you configure the AppCAT target Azure service as `Any`, you can switch between them to compare migration approaches and view service-specific recommendations. If you configure the target Azure service as a specific one, you can only see one Azure service in the dropdown list.
+
+:::image type="content" source="./media/list-azure-service-target-for-assessment-report.png" lightbox="./media/list-azure-service-target-for-assessment-report.png" alt-text="Screenshot of Visual Studio showing the GitHub Copilot app modernization assessment dashboard with Azure service target selection options.":::
+
+## Interpreting assessment report
+
+The assessment reports provide comprehensive analysis results to help you understand your application's readiness for Azure migration and modernization. This section guides you through the report structure and helps you interpret the findings to make informed migration decisions.
+
+### Report structure overview
+
+The assessment report consists of several key sections:
+
+- **Application Information**: Basic information about your application including project numbers, frameworks, build tools, and target Azure service.
+- **Issue Summary**: Overview of migration issues categorized by domain with criticality percentages.
+- **Issues**: Provides a concise summary of all issues that require attention.
+
+:::image type="content" source="./media/assessment-report-dashboard.png" lightbox="./media/assessment-report-dashboard.png" alt-text="Screenshot of Visual Studio showing the GitHub Copilot app modernization assessment report dashboard.":::
+
+#### Issues
+
+The issues section provides a categorized list of various aspects of Cloud Readiness that you need to address to successfully migrate the application to Azure. The following tables describe the `Domain` and `Criticality` values:
+
+| Domain              | Description                                                                             |
+|---------------------|-----------------------------------------------------------------------------------------|
+| **Cloud Readiness** | Evaluates app dependencies to suggest Azure services and ensure cloud-native readiness. |
+
+| Criticality         | Description                                                   |
+|---------------------|---------------------------------------------------------------|
+| **Mandatory**       | Issues that must be fixed for migration to Azure.             |
```

Comment:
Do we reveal informational level as well? if so we need add it here.

Reply:
no, we don't